lib/fluent/system_config.rb

```diff
@@ -55,6 +55,20 @@ class SystemConfig
     config_section :log, required: false, init: true, multi: false do
       config_param :format, :enum, list: [:text, :json], default: :text
       config_param :time_format, :string, default: '%Y-%m-%d %H:%M:%S %z'
+      config_param :rotate_age, default: nil do |v|
+        if %w(daily weekly monthly).include?(v)
```

Comment:
It seems better that this list is placed in lib/fluent/log.rb and `ROTATE_AGE` in lib/fluent/command/fluentd.rb refers same one.

Reply:
Ah, It should be,  Thanks!

Reply:
Fixed